README.md

```diff
@@ -7,10 +7,15 @@
 
 
 ## Built With
-
+ 
 - HTML & CSS
 - Linters
 
+## Getting Started
+
+To see the website click on the link below. 
+
+https://mugiira-kim.github.io/Portfolio-mobileskeleton/
```

Comment:
- [ ] [OPTIONAL] This is not the right syntax for adding a live demo link because it doesn't give the user a clickable link. I kindly suggest you do it like below

`[Live-Demo](https://mugiira-kim.github.io/Portfolio-mobileskeleton/)`